rustls/src/keylogfile.rs

```diff
@@ -0,0 +1,151 @@
+use crate::KeyLog;
+use std::env;
+use std::fs::{File, OpenOptions};
+use std::io;
+use std::io::Write;
+use std::path::PathBuf;
+use std::sync::Mutex;
+
+#[cfg(feature = "logging")]
+use crate::log::warn;
```

Comment:
Ideally group `crate` imports together, and separate them with an empty line from `std` imports (common style is to have `std` last rather than first).